### DISCUSSION TITLE
Can't log into browser extension - invalid request url path

### DISCUSSION THREAD

**Discussion by dominikheinz** [dani-garcia/vaultwarden#2327] [Q&A]
### Subject of the issue
Firstly, I can't pinpoint if this issue is related to the official bitwarden browser extension, or if its a problem that is caused because of the vaultwarden backend.
However, since this issue appeared in multiple browsers, and in different versions of the browser extension, I am guessing it might be related to the vaultwarden backend.
Essentially, I can not log into vaultwarden using the bitwarden browser extension anymore - password is obviously correct. Log directly via the web interface works just fine. Of course I have also set my set the server URL in the extension `https://something.something.com/`. When trying to log in on the browser extension, I get an error popup: "Invalid request url path".

I experienced this behaviour on chromium and on firefox. It used to work on a few days ago, but after reinstalling chromium, and using the same version of the browser extension, I run into this problem?? Again - this used to work just fine/

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.24.0
* Web-vault version: v2.25.1
* Running within Docker: true (Base: Debian)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: false
* HTTPS Check: false
* Database type: SQLite
* Database version: 3.35.4
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

**Environment settings which are overridden:** 

```json
{
  "_duo_akey": null,
  "_enable_duo": false,
  "_enable_email_2fa": false,
  "_enable_smtp": true,
  "_enable_yubico": true,
  "_ip_header_enabled": true,
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "data/attachments",
  "authenticator_disable_time_drift": false,
  "data_folder": "data",
  "database_max_conns": 10,
  "database_url": "****/**.*******",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "****://*********",
  "domain_origin": "****://*********",
  "domain_path": "",
  "domain_set": false,
  "duo_host": null,
  "duo_ikey": null,
  "duo_skey": null,
  "email_attempts_limit": 3,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 5 * * * *",
  "emergency_request_timeout_schedule": "0 5 * * * *",
  "enable_db_wal": true,
  "extended_logging": true,
  "helo_name": null,
  "hibp_api_key": null,
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "data/icon_cache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_org_name": "Vaultwarden",
  "invitations_allowed": true,
  "ip_header": "X-Real-IP",
  "job_poll_interval_ms": 30000,
  "log_file": null,
  "log_level": "Info",
  "log_timestamp_format": "%Y-%m-%d %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "",
  "password_iterations": 100000,
  "reload_templates": false,
  "require_device_email": false,
  "rsa_key_filename": "data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sends_allowed": true,
  "sends_folder": "data/sends",
  "show_password_hint": false,
  "signups_allowed": true,
  "signups_domains_whitelist": "",
  "signups_verify": false,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": false,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": null,
  "smtp_debug": false,
  "smtp_explicit_tls": false,
  "smtp_from": "",
  "smtp_from_name": "Vaultwarden",
  "smtp_host": null,
  "smtp_password": null,
  "smtp_port": 587,
  "smtp_ssl": true,
  "smtp_timeout": 15,
  "smtp_username": null,
  "templates_folder": "data/templates",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_syslog": false,
  "user_attachment_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "websocket_address": "0.0.0.0",
  "websocket_enabled": true,
  "websocket_port": 3012,
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```
</details>

* Other relevant details:
I am using nginx proxy manager, since the pi that is running vaultwarden is only exposed in my local network, so my URL maps to something like 192.168.178.xxx - but that never caused any issues.

### Steps to reproduce
I am honestly not sure, install the browser extension and try to connect to your local vaultwarden instance.

### Expected behaviour
Sign in should work normally.

### Actual behaviour
![image](https://user-images.githubusercontent.com/32770080/155174665-9901fa09-0627-4f50-aeaf-5ce65fc7fb8d.png)

**Comment by BlackDex** [comment]
I see that the domain seems to be configured incorrectly:
> Domain Configuration Check: false

Please double check that you have configured this as it should be.

**Reply by dominikheinz** [reply]
You mean the email configuration - or the domain which is used to host the vaultwarden instance?

**Reply by BlackDex** [reply]
Not sure where you see email here. But i mean the DOMAIN variable.
That isn't matching what you have in your browser it looks like
